### PR TITLE
Release 0.15

### PR DESCRIPTION
**Pull request by palango**
Release preparation PR

### CODE REVIEW

File: raiden-dapp/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "serve:no-logging": "export NODE_ENV=nologging && vue-cli-service serve"
   },
   "dependencies": {
-    "core-js": "^3.8.3",
```
Comment (palango) [inline raiden-dapp/package.json:21]: Any ideas why this happened?

Comment (andrevmatos) [reply raiden-dapp/package.json:21]: Weird.. maybe some leftover from a `yarn install` or alike? You sure you started the process with a clean environment?

File: raiden-dapp/CHANGELOG.md [hunk 2]
```diff
@@ -541,7 +543,8 @@
 - Add link to privacy policy.
 - Add basic transfer screen.
 
-[unreleased]: https://github.com/raiden-network/light-client/compare/v0.14.0...HEAD
+[unreleased]: https://github.com/raiden-network/light-client/compare/v0.15.0...HEAD
```
Comment (andrevmatos) [inline raiden-dapp/CHANGELOG.md:546]: These cases always get messed up.. nitpick though
```suggestion
[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.15.0...HEAD
```

**Review by andrevmatos — APPROVED** [review]
lgtm

File: raiden-dapp/package.json (hunk 1 above)
Comment (palango) [reply raiden-dapp/package.json:21]: Yeah, but the `prepare-release` script even runs `yarn install`

Comment (andrevmatos) [reply raiden-dapp/package.json:21]: Please, do a `serve` and also a prod `build` to check everything is fine without it. If yes, let it be.

Comment (palango) [reply raiden-dapp/package.json:21]: Ah, it's just duplicated in line https://github.com/raiden-network/light-client/pull/2535/files#diff-45d4a245bf7444d71c7e69b3bb96525d141aa212eeb829ddb643575ded0f9cfbR22

**Pull request merged**